Formula/geocouch.rb

```diff
@@ -1,11 +1,10 @@
 class Geocouch < Formula
```

Comment:
Okay.

I think full removal instead of boneyard in this case, since the formula will no longer be compatible with `couchdb` and is useless on its own. It'll still be there in the repo history if people really need it.

Amended to remove `geocouch`.